src/plugins/saved_objects_management/server/routes/find.test.ts

```diff
@@ -0,0 +1,161 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Any modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import { registerFindRoute } from './find';
+import { ISavedObjectsManagement } from '../services';
+import { IRouter } from 'src/core/server';
+
+describe('Find route', () => {
+  let router: jest.Mocked<IRouter>;
+  let managementService: jest.Mocked<ISavedObjectsManagement>;
+  let context: any;
+  let req: any;
+  let res: any;
+
+  beforeAll(() => {
+    router = {
+      get: jest.fn(),
+      handleLegacyErrors: jest.fn((handler) => handler),
+    } as any;
+
+    // Mock management service
+    managementService = {
+      isImportAndExportable: jest.fn().mockReturnValue(true),
+      getDefaultSearchField: jest.fn().mockReturnValue('title'),
+      getIcon: jest.fn().mockReturnValue('icon'),
+      getTitle: jest.fn().mockReturnValue('title'),
+      getEditUrl: jest.fn().mockReturnValue('/edit/mock'),
+      getInAppUrl: jest.fn().mockReturnValue('/api/mock'),
+      getNamespaceType: jest.fn().mockReturnValue('mockType'),
+    };
```

Comment:
We can leverage existing mocks from:
- https://github.com/opensearch-project/OpenSearch-Dashboards/blob/8a9666444b49c39a9c03aaf7c0af3a7c111564d8/src/core/server/http/http_service.mock.ts
- https://github.com/opensearch-project/OpenSearch-Dashboards/blob/d7004dc5b0392477fdd54ac66b29d231975a173b/src/plugins/saved_objects_management/server/services/management.mock.ts

to construct these mocks.

Reply:
Done